charts/gateway-operator/charts/kic-crds/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v2
+name: kic-crds
+version: 0.0.0
```

Comment:
Since this sub-chart is directly referenced through the directory, as long as we update this sub-chart, it will be automatically updated when installed via source code.

Reply:
Most installs aren't from a source checkout though, they're from the published registry bundle, and those include the specific version available at release time.